armi/reactor/components/component.py

```diff
@@ -964,12 +978,14 @@ def getThermalExpansionFactor(self, Tc=None, T0=None):
             :id: I_ARMI_COMP_EXPANSION0
             :implements: R_ARMI_COMP_EXPANSION
 
-            The dimensions of geometric information for components is heavily dependent
-            on the temperature of the component. This method will retrieve the thermal
-            expansion factor which accounts for the temperature's impact on the
-            component. Each material has unique thermal expansion properties. Therefore,
-            this method interrogates the material which comprises this component for its
-            thermal expansion factor.
+            This method enables the calculation of the thermal expansion factor
+            for a given material. If the material is solid, the difference
+            between T0 and Tc is used to calculate the thermal expansion
+            factor. If a solid material does not have a linear expansion factor
+            defined and the temperature difference is greater than
+            :py:attr:armi.reactor.components.component.Component._TOLERANCE, an
```

Comment:
```suggestion
            :py:attr:`armi.reactor.components.component.Component._TOLERANCE`, an
```
the ticks are needed for sphinx to link things right